elide-example-models/src/main/java/com/yahoo/elide/models/generics/Employee.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.models.generics;
+
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.models.BaseId;
+
+import javax.persistence.Entity;
+
+/**
+ * Helper class to test parameterized subclass/superclass hierarchies.
+ */
+@Include(rootLevel = true)
+@Entity
+public class Employee extends BaseId {
```

Comment:
I really like this package. Can we add a README.md that contains a brief description? To me this looks like a more complete model rather than what we have today (a bunch of disparate models intended to test specific functionality)